models/shopify__refunds.sql

```diff
@@ -0,0 +1,37 @@
+with refunds as (
+    select
+        refund_id,
+        note,
+        restock,
+        user_id,
+        order_id,
+        shop_url,
+        created_at_timestamp,
+        processed_at_timestamp
+
+    from {{ ref('stg_shopify_refunds') }}
+),
+
+orders as (
+    select *
+    from {{ ref('stg_shopify_orders') }}
+),
+
+final_refunds as (
+
+    select
+        refunds.refund_id,
+        refunds.note,
+        refunds.restock,
+        refunds.user_id,
+        refunds.order_id,
+        refunds.shop_url,
+        refunds.created_at_timestamp,
+        refunds.processed_at_timestamp,
+        orders.total_price
```

Comment:
As mentioned about I don't like this name since you get partial refunds

Reply:
same as above. Changed to sum(order_refunds_refund_line_items.subtotal) and name changed to total_refund_amount

Reply:
im not seeing this change?

Reply:
these 2 won't show up here. This is outdated. You need to look at the source code